.gitignore

```diff
@@ -1,2 +1,4 @@
 data/
-Data/
\ No newline at end of file
+Data/
+
+utils/__pycache__
```

Comment:
updated the .gitignore. This will be present in the datasets branch.

Reply:
Throws a really long list of errors when using the conda create env command. Attaching a text file that describes the error.

[error.txt](https://github.com/ms2892/advanced-ml/files/10795726/error.txt)